### PR TITLE
Support CF.MEXISTS command

### PR DESCRIPTION
**Pull request by doumdoum**

### CODE REVIEW

File: src/main/java/redis/clients/jedis/UnifiedJedis.java [hunk 1]
```diff
@@ -3922,6 +3923,9 @@ public boolean bfExists(String key, String item) {
 
   @Override
   public List<Boolean> bfMExists(String key, String... items) {
+    if (items.length == 0) {
+      return Collections.emptyList();
+    }
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/UnifiedJedis.java:3928]:
```suggestion
```
We don't do this in Jedis.

File: src/main/java/redis/clients/jedis/UnifiedJedis.java [hunk 2]
```diff
@@ -3975,6 +3979,15 @@ public boolean cfExists(String key, String item) {
     return executeCommand(commandObjects.cfExists(key, item));
   }
 
+  @Override
+  public List<Boolean> cfMExists(String key, String... items) {
+    if (items.length == 0) {
+      return Collections.emptyList();
+    }
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/UnifiedJedis.java:3986]:
```suggestion
```
We don't do this in Jedis.

File: src/test/java/redis/clients/jedis/modules/bloom/CuckooTest.java [hunk 3]
```diff
@@ -180,12 +181,25 @@ public void testInsertNxNoCreate() {
...
+
+  @Test
+  public void testNoItemMExists() {
+    assertEquals(Collections.emptyList(), client.cfMExists("cuckoo17"));
+  }
+
+  @Test
+  public void testMixedItemsMExists() {
+    client.cfInsert("cuckoo17", "foo");
+    assertEquals(Arrays.asList(true, false), client.cfMExists("cuckoo17", "foo", "bar"));
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/modules/bloom/CuckooTest.java:202]:
```suggestion
    client.cfInsert("cuckoo18", "foo");
    assertEquals(Arrays.asList(true, false), client.cfMExists("cuckoo18", "foo", "bar"));
```

File: src/main/java/redis/clients/jedis/CommandObjects.java [hunk 4]
```diff
@@ -3567,6 +3567,11 @@ protected CommandArguments commandArguments(ProtocolCommand command) {
     return new CommandObject<>(commandArguments(CuckooFilterCommand.EXISTS).key(key).add(item), BuilderFactory.BOOLEAN);
   }
 
+  public final CommandObject<List<Boolean>> cfMExists(String key, String... items) {
+    return new CommandObject<>(commandArguments(CuckooFilterCommand.MEXISTS).key(key).
+            addObjects((Object[]) items), BuilderFactory.BOOLEAN_LIST);
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/CommandObjects.java:3572]:
```suggestion
    return new CommandObject<>(commandArguments(CuckooFilterCommand.MEXISTS).key(key)
        .addObjects((Object[]) items), BuilderFactory.BOOLEAN_LIST);
```

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/UnifiedJedis.java (hunk 1 above)
Comment (doumdoum) [reply src/main/java/redis/clients/jedis/UnifiedJedis.java:3928]: Hi @sazzad16, varargs for items in function signature implicitly assumes 0 or more but redis expects 1 or more so we have to account for the 0 length (or we have to tell in documentation that 1 or more items are expected but i'd rather stick to regular understanding of function signature).

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/UnifiedJedis.java:3928]: @doumdoum `bfMExists` is not a simple straightforward word. The user who is calling bfMExists(), we assume that s/he has some idea about `BF.MEXISTS` command. We also take that documentation provided by Redis, e.g. https://oss.redis.com/redisbloom/Bloom_Commands/#bfmexists, is our prime documentation. Finally, we try to keep it as close to plain command as possible.

Comment (doumdoum) [reply src/main/java/redis/clients/jedis/UnifiedJedis.java]: @sazzad16 i can't agree with you: as a user of jedis i should not need to know redis protocol details, i should only rely on jedis API. Anyway i've remove the checks from the PR according to your proposal.

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**